docs/requirements.txt

```diff
@@ -2,8 +2,8 @@
 
 commonmark==0.9.1
 recommonmark==0.6.0
-Sphinx==3.3.0
-sphinx-rtd-theme==0.5.0
+Sphinx
```

Comment:
We should fixate all requirements (including below)

Reply:
borrowing it from monai.. they have removed the same..  https://github.com/Project-MONAI/MONAI/pull/2657/files#diff-2c9c11d26b09b8afde329980309d967121543a456e4592c76886a20b5cf56c90

Reply:
I don't think this is a good idea. That includes for the lines below